### PR TITLE
docs(tf): add authz to named profiles

### PR DESCRIPTION
**Pull request by 317brian**

### CODE REVIEW

**Review by aimeeu — APPROVED** [review]
LGTM; waiting for approval from tech reviewer

File: content/en/docs/spinnaker/terraform-enable-integration.md
Comment (KathrynLewis) [inline content/en/docs/spinnaker/terraform-enable-integration.md]: > do not see it as an option in Deck.

And maybe add that the stage will fail if they try to run a stage with an unauthorized profile.

**Review by KathrynLewis — APPROVED** [review]
One small comment but looks great!

**Pull request merged**
